config/7.10.0/common.yaml

```diff
@@ -33,6 +33,7 @@ console:
               app: "[[.ApplicationName]]"
               application: "[[.ApplicationName]]"
               service: "[[.ApplicationName]]-[[.Console.Name]]"
+              cluster: "jgrp.k8s.[[.ApplicationName]].[[.Console.Name]]"
```

Comment:
please move all config change to the `config/7.11.0` configs instead. e.g. `config/7.11.0/common.yaml`

Reply:
Done